config.tf

```diff
@@ -74,6 +74,13 @@ locals {
         mount_path = "${var.local_cache_dir}"
         host_path = "${var.local_cache_dir}"
     %{~endif~}
+    %{~for name, config in var.build_job_hostmounts~}
+      [[runners.kubernetes.volumes.host_path]]
+        name = "${name}"
+        mount_path = "${config.path}"
```

Comment:
@chickenandpork  I'm thinking of cases where someone wants to mount a host path to a different path in the container. Would it be better to make this `config.container_path` and `config.host_path`

Reply:
yeah, that makes sense and is quick to add